pkg/ui/src/InventoryTable/InventoryTableRow.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+	import type { TableData } from './types';
+
+	export let rowData: TableData;
```

Comment:
Can flatten this into named props (have `isbn`, `title`, etc...instead of them being properties on `rowData`) That's easier to provide fallbacks for and more efficient if only one field changes (we don't rerender the entire row, only the field)